megameklab/src/megameklab/printing/PrintAero.java

```diff
@@ -245,7 +245,8 @@ protected void drawFluffImage() {
         if (null != fluffImage) {
             Element rect = getSVGDocument().getElementById(FLUFF_IMAGE);
             if (rect instanceof SVGRectElement) {
-                embedImage(fluffImage, (Element) rect.getParentNode(), getRectBBox((SVGRectElement) rect), true);
+                embedImage(fluffImage, (Element) rect.getParentNode(), getRectBBox((SVGRectElement) rect), true,
+                      "fluff-image");
```

Comment:
Rather than using literal strings in multiple places, we should make a constant for better maintainability.
Additionally, is there a good reason to use "fluff-image" here, when we have the existing FLUFF_IMAGE constant (aka "fluffImage") as well as "fluffimage" as used in the BLK files?